libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-overview.component.ts

```diff
@@ -14,39 +14,69 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
-import { LocalRouterStore } from '@ngworker/router-component-store';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  OnDestroy,
+} from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { map, Subject, takeUntil } from 'rxjs';
 import { LetModule } from '@rx-angular/template';
-import { map } from 'rxjs';
-
-import { WattSpinnerModule } from '@energinet-datahub/watt';
+import { TranslocoModule } from '@ngneat/transloco';
+import { WattBadgeModule, WattSpinnerModule } from '@energinet-datahub/watt';
+import { DhMeteringPointDataAccessApiStore } from '@energinet-datahub/dh/metering-point/data-access-api';
+import { LocalRouterStore } from '@ngworker/router-component-store';
 
 import { DhBreadcrumbScam } from './breadcrumb/dh-breadcrumb.component';
-import { DhMeteringPointDataAccessApiStore } from '@energinet-datahub/dh/metering-point/data-access-api';
 import { dhMeteringPointIdParam } from './routing/dh-metering-point-id-param';
+import { DhMeteringPointNotFoundScam } from './not-found/dh-metering-point-not-found.component';
+import { DhMeteringPointServerErrorScam } from './server-error/dh-metering-point-server-error.component';
+import { DhMeteringPointStatusBadgeScam } from './status-badge/dh-metering-point-status-badge.component';
+import { DhMeteringPointPresenter } from './dh-metering-point-overview.presenter';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'dh-metering-point-overview',
   styleUrls: ['./dh-metering-point-overview.component.scss'],
   templateUrl: './dh-metering-point-overview.component.html',
-  providers: [DhMeteringPointDataAccessApiStore, LocalRouterStore],
+  viewProviders: [
+    LocalRouterStore,
+    DhMeteringPointDataAccessApiStore,
+    DhMeteringPointPresenter,
+  ],
 })
-export class DhMeteringPointOverviewComponent {
+export class DhMeteringPointOverviewComponent implements OnDestroy {
+  private destroy$ = new Subject<void>();
+
+  meteringPointId$ = this.route.selectRouteParam<string>(
+    dhMeteringPointIdParam
+  );
   meteringPoint$ = this.store.meteringPoint$;
+  isLoading$ = this.store.isLoading$;
+  meteringPointNotFound$ = this.store.meteringPointNotFound$;
+  hasError$ = this.store.hasError$;
+  translationKeys$ = this.presenter.translationKeys$;
+
+  emDash = '—';
```

Comment:
Question: Should we move this to the translations or something? We are properly going to use this a lot of places